### PR TITLE
Finish screen

### PR DESCRIPTION
**Pull request by thetaPC**
Data is saved to Firebase

### CODE REVIEW

File: database.rules.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "rules": {
-    ".read": "false",
-    ".write": "false"
+    ".read": "auth != null",
```
Comment (Reshurum) [inline database.rules.json]: **Security**: We need to add rules so anonymous users cannot edit other user's forms.

File: firebase.json [hunk 2]
```diff
@@ -3,12 +3,6 @@
...
   },
   "hosting": {
-    "public": "build/default",
-    "rewrites": [
-      {
-        "source": "**",
-        "destination": "/index.html"
-      }
-    ]
+    "public": "public"
```
Comment (Reshurum) [inline firebase.json]: This should be changed back, polyer builds to `build/default`.

File: public/404.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Page Not Found</title>
+
+    <style media="screen">
+      body { background: #ECEFF1; color: rgba(0,0,0,0.87); font-family: Roboto, Helvetica, Arial, sans-serif; margin: 0; padding: 0; }
```
Comment (Reshurum) [inline public/404.html]: Don't squash css like this please.

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    <!-- include only the Firebase features as you need -->
+    <script defer src="/__/firebase/4.1.5/firebase-auth.js"></script>
+    <script defer src="/__/firebase/4.1.5/firebase-database.js"></script>
+    <script defer src="/__/firebase/4.1.5/firebase-messaging.js"></script>
+    <script defer src="/__/firebase/4.1.5/firebase-storage.js"></script>
+    <!-- initialize the SDK after all desired features are loaded -->
+    <script defer src="/__/firebase/init.js"></script>
+
+    <style media="screen">
+      body { background: #ECEFF1; color: rgba(0,0,0,0.87); font-family: Roboto, Helvetica, Arial, sans-serif; margin: 0; padding: 0; }
```
Comment (Reshurum) [inline public/index.html]: Don't squash css like this please.

File: public/404.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (Reshurum) [inline public/404.html]: I don't think this file was intended to be here?

File: src/my-view1.html [hunk 6]
```diff
@@ -50,7 +51,14 @@
         max-width: 250px;
       }
 
-    </style>
+</style>
```
Comment (Reshurum) [inline src/my-view1.html]: Fix indentation of style tag, vscode auto indent is buggy right now (it's been happening to me too).

File: src/my-view1.html [hunk 7]
```diff
@@ -220,6 +233,15 @@
             };
           }
         });
+
+        this.$.responsiveStepper.addEventListener('paper-stepper-completed', () => {
+          console.log('completed');
```
Comment (Reshurum) [inline src/my-view1.html]: Remove console.log before merging.

File: src/my-view1.html [hunk 8]
```diff
@@ -121,6 +129,8 @@
       <img src="images/confetti.svg">
       <h2>Let the world know on the social media!</h2>
     </div>
+
```
Comment (Reshurum) [inline src/my-view1.html]: Double newline is unnecessary.

File: public/index.html (hunk 4 above)
Comment (thetaPC) [inline public/index.html]: These files were generated by `firebase init` and not by us.

File: database.rules.json (hunk 1 above)
Comment (thetaPC) [inline database.rules.json]: Since users aren't allowed to edit their forms, do these rules need to be changed?

File: firebase.json [hunk 9]
```diff
@@ -3,6 +3,6 @@
     "rules": "database.rules.json"
   },
   "hosting": {
-    "public": "public"
+     "public": "build/default"
```
Comment (Reshurum) [inline firebase.json]: The routing information is still missing.

File: database.rules.json [hunk 10]
```diff
@@ -1,6 +1,6 @@
 {
   "rules": {
-    ".read": "auth != null",
-    ".write": "auth != null"
+    ".read": "false",
+    ".write": "true"
```
Comment (Reshurum) [inline database.rules.json]: With this rule, I can nuke the database.

**Pull request merged**